jenkinsci/managed-scripts-plugin
# [JENKINS-74119] Remove inline onClick handlers in `PowerShellBuildScript/config.jelly`

[JENKINS-74119](https://issues.jenkins.io/browse/JENKINS-74119)

Very similar to #30 

There is an onClick Handler on the `Add Argument` button however it looks like the functionality was previously removed. I removed it and it does not affect the functionality at all. [relevant commit](https://github.com/jenkinsci/managed-scripts-plugin/commit/f691a78cce9bcce34a9d7e6ab270321b4a76ecd7#diff-6331cf999f244b590f324032c048dc393bde1dee141477b2d48f17256f33a2e4)

### Testing done

After Changes:

https://github.com/user-attachments/assets/7cf9a63e-c99c-4021-b39a-6c0b68fa6c77

1. Go to Manage Jenkins -> Managed Files -> Add new config file
2. Select 'Managed Powershell File` -> Save and submit.
3. Create Freestyle Project -> In Build Steps select: `Execute Managed Powershell`
4. Select the script create above -> Here you will see the code changed:
  i. `view selected file`
  ii. `Add Argument`




### Submitter checklist
